tests/Unit/LoggerTraitTest.php

```
@@ -0,0 +1,230 @@
+
+    // Call various methods.
+    static::log('Message 1');
+
+    // Disable verbose mode.
+    static::loggerSetVerbose(FALSE);
+
+    // Call methods again.
+    static::log('Message 2');
+
+    // Verify verbose mode changes work correctly.
+    $this->assertTrue(TRUE);
```

PHPStan (2.1.15)

Call to method PHPUnit\Framework\Assert::assertTrue() with true will always evaluate to true.


(method.alreadyNarrowedType)